zebra/zebra_rib.c

```diff
@@ -1479,15 +1479,12 @@ static bool rib_route_match_ctx(const struct route_entry *re,
 		    (re->instance == dplane_ctx_get_instance(ctx))) {
 			result = true;
 
-			/* We use an extra test for statics, and another for
-			 * kernel routes.
+			/*
+			 * We use different tests for kernel and for
+			 * connected routes.
 			 */
-			if (re->type == ZEBRA_ROUTE_STATIC &&
```

Comment:
did you want to remove this test for the "update" clause above?

Reply:
ah I should shouldn't I  give me a bit